src/pyproject.toml

```diff
@@ -83,6 +83,7 @@ urllib3 = "1.26.18"
 aiohttp = "3.9.5"
 cryptography = "42.0.5"
 requests-mock = "^1.12.1"
+django-simple-history = "^3.7.0"
```

Comment:
I suggest to target a specific version instead of a range.

Reply:
Good catch, I thought I did